### DISCUSSION TITLE
Alignment of protruding pixels when converting to monochrome

### DISCUSSION THREAD

**Discussion by Ostrowin** [ImageMagick/ImageMagick#4787] [Help]
I need to convert .png image with some font awesome to monochrome. I am using cmd 
`convert image.png -colors 2 -type palette BMP:imagex.bmp`
and the result is bad. There are a lot of pixels out of the word or icon. I know i am converting to monochrome, but do anyone know a solution for smoothing image to prevent outline pixels?
![pngicocn](https://user-images.githubusercontent.com/35689589/151755863-1ed1c27e-74a3-4ea5-8b0c-7530c380ac07.png)
![bmpicno](https://user-images.githubusercontent.com/35689589/151755888-c595b295-2920-41ca-ad79-9f4af4bf6e16.png)
![wordpng](https://user-images.githubusercontent.com/35689589/151755927-b8f775c3-e1a9-44ed-957a-c0c09f052af2.png)
![bmpword](https://user-images.githubusercontent.com/35689589/151755950-bcf69615-64d6-47ff-9d0c-185ac2404429.png)

**Comment by Ostrowin** [comment]
The answer can be -auto-threshold OTSU option

**Comment by snibgo** [comment]
Why are your outputs larger than your inputs?

Please show the actual commands you used.

You use dithering. You might prefer not to: "+dither" before "-colors".

**Reply by Ostrowin** [reply]
now i am using cmd:
`convert image.png -auto-threshold OTSU -colors 2 -type palette BMP:image.bmp`
Why souldn't I use +dither before -colors?

**Comment by snibgo** [comment]
I don't understand what problem you want to solve. The command you showed didn't make those results from those inputs.

**Comment by fmw42** [comment]
Perhaps what you want is:

`convert image.png -threshold 50% x1.png`


![x1](https://user-images.githubusercontent.com/13458218/152039526-0e3cbca0-054d-4c63-bea5-c5d5060e466c.png)
